packages/build/src/telemetry/main.js

```diff
@@ -2,15 +2,19 @@
 
 const { platform, version: nodeVersion } = require('process')
 
-const execa = require('execa')
+const got = require('got')
 const osName = require('os-name')
 const { v4: uuidv4 } = require('uuid')
 
 const { version } = require('../../package.json')
 const { roundTimerToMillisecs } = require('../time/measure')
 
-// Script file that will be responsible for making the actual telemetry request
-const REQUEST_FILE = `${__dirname}/request.js`
```

Comment:
This file should be removed too.

Reply:
True! Thanks 🙏

Reply:
Addressed via https://github.com/netlify/build/pull/2438/commits/88419ae366d83d272229226fa481ce119eb444bf